admin_manual/configuration/server/occ_command.rst

```diff
@@ -843,16 +844,26 @@ Groups containing the ``search-pattern`` string are listed. Matching is
 not case-sensitive. If you do not provide a search-pattern then all groups 
 are listed.
 
-This example lists groups containing the string finance:: 
+Options:
+
+::
+  
+  --output[=OUTPUT]  Output format (plain, json or json_pretty, default is plain) [default: "plain"]
+
+This example lists groups containing the string finance
```

Comment:
s/This example lists groups containing the string finance/This example lists groups containing the string "finance"./g